src/app/icons/spotify-with-text-icon/spotify-with-text-icon.html

```
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 559 168" preserveAspectRatio="xMidYMid meet">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)